containers/layouts/DraftTypeSelect.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
-import { Select } from 'react-components';
+import { Select, useNotifications } from 'react-components';
+import { DRAFT_TYPE } from 'proton-shared/lib/constants';
+
+const { NORMAL, PLAIN_TEXT } = DRAFT_TYPE;
 
 const DraftTypeSelect = ({ draftType, onChange, loading }) => {
     const options = [
-        { text: c('Option').t`Normal`, value: 'text/html' },
-        { text: c('Option').t`Plain Text`, value: 'text/plain' }
+        { text: c('Option').t`Normal`, value: NORMAL },
+        { text: c('Option').t`Plain Text`, value: PLAIN_TEXT }
     ];
+    const { createNotification } = useNotifications();
+
+    const handleChange = ({ target }) => {
+        if (target.value != draftType) {
```

Comment:
```suggestion
        if (target.value !== draftType) {
```

Reply:
I removed this line since it was superfluous, as `handleChange` is only called when the value changes.